Assets/MixedRealityToolkit/Utilities/Gltf/Schema/GltfAnimationChannelPath.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Gltf.Schema
+{
+    /// <summary>
+    /// The name of the node's TRS property to modify, or the weights of the Morph Target it istantiates.
```

Comment:
Can we get an expansion of "TRS" to help customers better understand what it is?

Reply:
This is a direct copy from khronos gltf schema. I'd rather stay consistent with their documentation